tests/test_vis.py

```diff
@@ -20,9 +20,40 @@ def test_vis_mds_output_shape_corresponds_to_inputs(self):
         rdms = rsr.RDMs(dissimilarities=dis,
                         dissimilarity_measure=mes,
                         descriptors=des)
-        mds_emb = rsv.vis.mds(rdms)
+        mds_emb = rsv.mds(rdms)
         self.assertEqual(mds_emb.shape, (8, 5, 2))
 
+    def test_vis_3d_mds_output_shape_corresponds_to_inputs(self):
+        dis = np.random.rand(8, 10)
+        mes = "Euclidean"
+        des = {'session': 0, 'subj': 0}
+        rdms = rsr.RDMs(dissimilarities=dis,
+                        dissimilarity_measure=mes,
+                        descriptors=des)
+        mds_emb = rsv.mds(rdms, dim=3)
+        self.assertEqual(mds_emb.shape, (8, 5, 3))
```

Comment:
The tests only test the dimensions of the output. Ideally they would also test the difference in the weighted case

Reply:
done.